procedures/unit-testing-reporting.ipf

```diff
@@ -5,6 +5,77 @@
 
 static Constant IP8_PRINTF_STR_MAX_LENGTH = 2400
 
+/// @brief Get the current date time in seconds since 1904-01-01 as a string
+static Function/S GetDateTime()
+	string msg
+	sprintf msg, "%.3f", DateTime
```

Comment:
`DateTime` has a millisecond precision and returns the number of seconds since an arbitrary date. I need to store the result of this function as string in a text wave (I would have used numbers but this is not possible). Therefore I need to convert this number in some way to a string. If I use `%f` as a format specifier it would return the same as `%.6f` which returns more digits than the supported precision of `DateTime`.

Reply:
I have changed the method name and updated the documentation. Also StopMSTimer(-2) is now used.